### PR TITLE
Use Core testing environment

### PR DESCRIPTION
**Pull request by fjvallarino**
@Rise-Vision/apps these are the changes I made for re enabling the test profile for Viewer. This PR is not necessarily meant to be merged; I just want to validate if the change set seems correct.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -62,7 +63,8 @@ gulp.task('test:e2e:core', ['test:webdrive_update'], factory.testE2EAngular({
 }));
 gulp.task('test:webdrive_update', factory.webdriveUpdate());
 gulp.task('test:e2e', (cb) => {
-  seq('config:e2e', 'build', 'server', 'test:e2e:core', 'server-close', cb);
+  cb();
+  //seq('config:e2e', 'build', 'server', 'test:e2e:core', 'server-close', cb);
```
Comment (alex-deaconu) [inline gulpfile.babel.js]: @fjvallarino e2e tests are disabled here. Thanks.

File: circle.yml [hunk 2]
```diff
@@ -31,8 +31,8 @@ deployment:
   staging:
     branch: /(feature|fix|chore).*/
     commands:
-      #- ./node_modules/.bin/cross-env NODE_ENV=dev npm run build
-      - ./node_modules/.bin/cross-env NODE_ENV=prod npm run build
+      - ./node_modules/.bin/cross-env NODE_ENV=dev npm run build
+      #- ./node_modules/.bin/cross-env NODE_ENV=prod npm run build
```
Comment (alex-deaconu) [inline circle.yml]: @fjvallarino I'm assuming this is used for staging. Just a reminder to revert this change before/if merging. Thanks.

File: gulpfile.babel.js (hunk 1 above)
Comment (fjvallarino) [inline gulpfile.babel.js]: @alex-deaconu yes, they were disabled since I wanted to get a version running and e2e tests were not passing. As I said in the PR's header, I don't necessarily mean to merge this version. I created the PR to discuss the config mechanism. This version does not modify Viewer itself, so e2e test cases were not supposed to fail because of my change.

**Comment by alex-deaconu** [comment]
@fjvallarino couple of comments. Otherwise looks good 👍 . Thanks.

File: gulpfile.babel.js (hunk 1 above)
Comment (alex-deaconu) [inline gulpfile.babel.js]: @fjvallarino understood. Thanks.

File: circle.yml (hunk 2 above)
Comment (alex-deaconu) [inline circle.yml]: @fjvallarino CS have always preferred to use the Production environment for Viewer testing. We stage branches to Production, and I would say, ideally try to keep the default version pointing to the Test environment. Thanks.

**Comment by tejohnso** [comment]
@fjvallarino can we close this old pr?

**Comment by alex-deaconu** [comment]
Francisco is away, so I'm answering for him.

@tejohnso yup. Good to close. Thanks!
